app/views/users/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>SHOW INFO OF ONE USER</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <div class='container'>
+      <div class='card'>
+        <img class='userpic' src= <%= @user.photo %> >
+        <div class='leftcard'>
+          <h2 class='username'><%= @user.name %></h2>
+          <p class='count'>Number of posts: <%= @user.posts_counter >= 0 ? @user.posts_counter : '0' %></p><br>
+        </div>
+      </div>
+      <div class='biodiv'>
+        <p>Bio</p>
+        <p><%= @user.bio %></p>
+      </div>
+      <% @posts.each do |post| %>
+        <div class='postcard'>
+          <p><%= post.title %></p>
+          <p><%= post.text.truncate(50, separator:' ') %></p>
+          <p class='count'>
+            Comments: <%= post.comments_counter != nil ? post.comments_counter : '0' %>
+            Likes: <%= post.likes_counter != nil ? post.likes_counter : '0' %>
+          </p>
+        </div>
+      <% end %>
+      <a class='button' href=<%= user_posts_url(@user.id)%>>See All Posts</a>
+    </div>
```

Comment:
- Okay 👍 
![Capture](https://user-images.githubusercontent.com/63972670/214708343-aba07c29-e147-4e49-b4ed-3497c8de16e8.JPG)

Reply:
If it is ok, what should I change?